src/components/Terminal.jsx

```diff
@@ -173,8 +168,16 @@ export default class Terminal extends React.Component {
       const args = input // ...and the rest can be used
 
       if (!this.props.noAutomaticStdout) {
-        if (!this.props.noHistory) this.pushToStdout(`${this.props.promptLabel || '$'} ${rawInput}`, rawInput)
-        else this.pushToStdout(`${this.props.promptLabel || '$'} ${rawInput}`)
+        this.pushToStdout([
```

Comment:
This block is indented wrong and has dangling commas, does not pass the linter phase.